configurations/rolling-upgrade-artifacts.yaml

```
@@ -0,0 +1,6 @@
+# workloads
+stress_before_upgrade: cassandra-stress write cl=QUORUM n=10000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
```

```suggestion
stress_before_upgrade: cassandra-stress write cl=QUORUM n=10000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -pop seq=1..10000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5
```

Don't mention the CS, so in enterprise it will be ICS.

---

Changed